Source/Core/Core/IOS/IOS.cpp

```diff
@@ -304,7 +304,7 @@ Kernel::Kernel(IOSC::ConsoleType console_type) : m_iosc(console_type)
   ASSERT(m_fs);
 
   std::lock_guard lock(m_device_map_mutex);
```

Comment:
I don't think we need to lock this mutex anymore

Reply:
Mhm, I suppose it doesn't make much sense to lock this in the constructor, yeah.

Reply:
In fact... is this mutex needed at all? It's only locked in the constructor, destructor, and a single function (`GetDeviceByName()`).